src/components/Displayresults.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+
+class Displayresults extends React.Component {
+  constructor(){
+    super()
+    this.fetchFilmInfo = this.fetchFilmInfo.bind(this)
+    this.handleClick = this.handleClick.bind(this)
+    this.state ={
+        info: {},
+        MovieInfoHeading: ''
+        
+    }
+  }
+
+  handleClick(){
+
+  }
+
+  fetchFilmInfo(){
+    fetch(`http://www.omdbapi.com/?apikey=8d5ab09&i=${this.props.movie.imdbID}`)
+    .then(response => response.json())
+    .then(body => 
+      this.setState({
```

Comment:
It would be better to store the returned movie object in state and generate the strings to display to user in render method